src/components/asset-list/RecyclerAssetList2/FastComponents/FastBalanceCoinRow.tsx

```diff
@@ -0,0 +1,320 @@
+import React, { useCallback, useRef } from 'react';
+import { StyleSheet, View } from 'react-native';
+import { CoinIconIndicator } from '../../../../components/coin-icon';
+import { Icon } from '../../../../components/icons';
+import { ButtonPressAnimation } from '../../../animations';
```

Comment:
`@rainbow-me/....`

Reply:
what do you mean exactly? we don't have any specific rainbow-me imports for these